PhysicsTools/PatAlgos/interface/MuonMvaIDEstimator.h

```diff
@@ -0,0 +1,37 @@
+#ifndef __PhysicsTools_PatAlgos_MuonMvaIDEstimator__
```

Comment:
```suggestion
#ifndef PhysicsTools_PatAlgos_MuonMvaIDEstimator_h
```
http://cms-sw.github.io/cms_coding_rules.html rule 4.1